apps/admin/frontend/src/app.test.tsx

```diff
@@ -482,68 +427,33 @@ test('L&A (logic and accuracy) flow', async () => {
   );
 });
 
-test('L&A features are available after test results are loaded', async () => {
-  const hardware = MemoryHardware.buildStandard();
-  const backend = new ElectionManagerStoreMemoryBackend({
-    electionDefinition: eitherNeitherElectionDefinition,
-    castVoteRecordFiles: await CastVoteRecordFiles.empty.add(
-      EITHER_NEITHER_CVR_TEST_FILE,
-      eitherNeitherElectionDefinition.election
-    ),
-  });
-  renderRootElement(<App hardware={hardware} />, {
-    apiClient: mockApiClient,
-    backend,
-  });
-
-  await authenticateAsElectionManager(
-    mockApiClient,
-    eitherNeitherElectionDefinition
-  );
-
-  // Confirm that test results are loaded
-  userEvent.click(screen.getByText('Tally'));
-  await screen.findByText('Currently tallying test ballots.', { exact: false });
-  expect(screen.getByTestId('total-cvr-count').textContent).toEqual('100');
-
-  // Confirm that L&A materials are available
-  userEvent.click(screen.getByText('L&A'));
-  screen.getByText('List Precinct L&A Packages');
-  screen.getByText('Print Full Test Deck Tally Report');
-});
-
-test('printing ballots and printed ballots report', async () => {
-  const backend = new ElectionManagerStoreMemoryBackend({
-    electionDefinition: eitherNeitherElectionDefinition,
-  });
-
-  const printer = fakePrinter();
+test('printing ballots', async () => {
```

Comment:
printed ballots report moved to a different test below focusing on rending information in the reports screen.